src/clarity-angular/popover/common/_popover.clarity.scss

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+// This software is released under MIT license.
+// The full license information can be found in LICENSE in the root directory of this project.
+
+@include exports('popover.clarity') {
+    .is-off-screen {
+        position: fixed !important;
+        height: 1px !important;
+        width: 1px !important;
+        top: -1px !important;
+        padding: 0 !important;
+        margin: 0 0 -1px 0 !important;
+        border: none !important;
+    }
```

Comment:
Hum, you forgot `overflow: hidden`. Maybe it's applied somewhere else too, but it's so critical to this off-screen class that it absolutely needs to be here too.

Reply:
👍